diagnose-codespan-reporting/README.md

```diff
@@ -0,0 +1,5 @@
+# `codespan-reporting` layout for Diagnose
+
+This is an implementation of a custom layout for Diagnose, heavily inspired by [`codespan-reporting`](https://github.com/brendanzab/codespan).
+
+TODO: add example rendering
```

Comment:
I see this has not yet been replaced.
Should I add examples myself (for consistency with other screenshots)?

Reply:
Ah, I totally forgot about this. If you could point me to the code responsible for generating the example, I can add the screenshots here. By the way, I think SVG screenshots (as shown in my previous comments in this PR) might have better display quality (and also smaller in size); I can replace other screenshots to use SVG format if you want (I already have the tool installed so it might be more convenient to do it on my side).